resources/idlharness.js

```diff
@@ -1422,7 +1422,7 @@ IdlInterface.prototype.test = function()
         {
             subsetTestByKey(this.name, test, function() {
                 assert_false(this.name in self);
-            }.bind(this), this.name + " interface: existence and properties of interface object");
+            }.bind(this), this.name + " interface: nonexistence of interface object");
```

Comment:
Hmm, I guess the reason for this is that we then always have a subtest with this name. Would it be sufficient to add a message to the `assert_false`?